src/clients/cache/momento/commands/set_if_present_and_not_equal.rs

```diff
@@ -0,0 +1,42 @@
+use super::*;
+use ::momento::cache::SetIfPresentAndNotEqualResponse;
+
+pub async fn set_if_present_and_not_equal(
+    client: &mut CacheClient,
+    config: &Config,
+    cache_name: &str,
+    request: workload::client::SetIfPresentAndNotEqual,
+) -> std::result::Result<(), ResponseError> {
+    SET_IF_PRESENT_AND_NOT_EQUAL.increment();
+
+    match timeout(
+        config.client().unwrap().request_timeout(),
+        client.set_if_present_and_not_equal(
+            cache_name,
+            (*request.key).to_owned(),
+            (*request.new_value).to_owned(),
+            (*request.old_value).to_owned(),
+        ),
+    )
+    .await
+    {
+        Ok(Ok(r)) => match r {
```

Comment:
Same here as with my comment above. record_result + if let might make this cleaner)